wgpu/src/backend/webgpu.rs

```diff
@@ -374,6 +374,7 @@ fn map_stencil_state_face(desc: &wgt::StencilFaceState) -> webgpu_sys::GpuStenci
     mapped
 }
 
+
```

Comment:
Could you remove the newlines and add back the line that was removed in this file?

Reply:
Should be fixed now